### ISSUE TITLE
Replace Docker tag with Crane tag or Crane cp.

### ISSUE THREAD

**Issue by kristof-mattei** [semantic-release-plus/semantic-release-plus#122]
https://github.com/semantic-release-plus/semantic-release-plus/blob/e782b5e0c6f1e52c4e7a5e26a81b3e7daf98b538/packages/plugins/docker/src/lib/docker-utils.ts#L24-L34

I am trying to use this project and I noticed that under certain conditions, such as pull existing tag, retag, and push new tag, a new digest is created. 

I still haven't figured out WHEN it actually happens, for example, if you pull an existing tag, add 2 new tags, and push both, both of the new ones get the same new digest. 

I have been able to mitigate this by using `crane` in another project. `crane cp`, or `crane tag` basically allows you to retag but the digest doesn't change. An added bonus is `crane tag` doesn't pull down the image, like specified here: 

https://stackoverflow.com/questions/64779312/tagging-and-pushing-docker-image-changes-digest

The downside is the dependency on `crane`. We can run it through `docker`, but it requires just some setup to make sure it can log-in.

**Comment by kristof-mattei** [comment]
@JoA-MoS Okay, this is awkward. 

I created a whole POC to show you, and I even started integrating a switch to use `crane` in your software for start to help out.

However... I cannot reproduce it...

This makes me wonder if `ghcr.io` got updated between October 19th and now. I filed a ticket with GitHub technical support. & keep you posted.

**Comment by JoA-MoS** [comment]
@kristof-mattei No, worries, and I appreciate anyone's help, I also appreciate the fact that you took the time to try and reproduce rather than just creating as solution. I am not sure if ghcr.io was updated or not but I did run through most of the scenarios on my test repo here https://github.com/JoA-MoS/srp-docker-example/pkgs/container/srp-docker-example. I also didn't release the current version 3.x.x until the 21st of October, so if you saw the issue prior to then you might have seen an issue with the previous version. There was a version I think it only made it to the beta channel that may not have included the add-channel hook.

Anyways, thanks for using the plugin and if you find something that may have caused the issue let me know and we can reopen this issue.
